adafruit_wiznet5k/adafruit_wiznet5k_socket.py

```diff
@@ -160,6 +185,34 @@ def listen(self, backlog=None):
         _the_interface.socket_listen(self.socknum, self._listen_port)
         self._buffer = b""
 
+    def accept(self):
```

Comment:
Dont need to say "mimic..", change to the docstring for the CPython API (https://docs.python.org/3/library/socket.html#socket.socket.accept)